doc/examples/app_configs.rst

```diff
@@ -0,0 +1,77 @@
+Application Configs
+^^^^^^^^^^^^^^^^^^^
+
+Application ``my_app`` has two build configs, which in this trivial case change the value
```

Comment:
We should probably stick to the naming scheme we've settled on for the examples .i.e. app_my_app or similar